src/rpc/util.cpp

```diff
@@ -475,6 +481,20 @@ std::string RPCExamples::ToDescriptionString() const
     return m_examples.empty() ? m_examples : "\nExamples:\n" + m_examples;
 }
 
+template <> RPCDefaultParam<bool>::operator bool() const { return value.isNull() ? defaults : value.get_bool(); }
+template <> RPCDefaultParam<int>::operator int() const { return value.isNull() ? defaults : value.get_int(); }
+template <> RPCDefaultParam<std::string>::operator std::string() const { return value.isNull() ? defaults : value.get_str(); }
+template <> RPCParam<UniValue>::operator UniValue() const { return value; }
+template <> RPCParam<bool>::operator bool() const { return value.get_bool(); }
+template <> RPCParam<int>::operator int() const { return value.get_int(); }
+template <> RPCParam<std::string>::operator std::string() const
+{
+    if (!value.isNull()) return value.get_str();
+    CHECK_NONFATAL(arg.m_fallback.which() == 1);
```

Comment:
```suggestion
    CHECK_NONFATAL(arg.m_fallback.type() == typeid(std::string));
```
and the two other places within this file that call `m_fallback.which()`. However, you're not changing those other two places in this PR, so if you don't want to change them (personally I would favor doing so, as this is much easier to understand), you may want to not take this suggestion.